src/parity_testing/auth/testSuites/signIn/Test_that_Device_SRP_signIn_invokes_proper_cognito_request_and_returns_success.json

```diff
@@ -9,13 +9,12 @@
                 "apiName": "initiateAuth",
                 "responseType": "success",
                 "response": {
-                    "challengeName": "PASSWORD_VERIFIER",
```

Comment:
Using valid response so that sign in could actually succeed.

Reply:
Don't you need PASSWORD_VERIFIER as challenge to call respondToAuthChallenge with this challenge?

Reply:
@div5yesh Check below, we use it.. Just changed `challengeName` to `PascalCase`